app/assets/javascripts/hubstats/application.js

```diff
@@ -14,21 +14,28 @@
 //= require jquery_ujs
 //= require select2
 //= require hubstats/bootstrap
+//= require bootstrap-datepicker
 //= require_tree .
 
-
 $(document).ready( function() {
-  setTimespan()
+  setDateRange()
 });
 
-function setTimespan() {
+// $(document).on("focus", "[data-behaviour~='datepicker']", function(e){
```

Comment:
You should consider deleting this. I'm not a fan of leaving code commented out. We can always bring it back with git.